components/tyk-pump/templates/deployment-pmp.yaml

```diff
@@ -118,6 +119,17 @@ spec:
                 name: {{ include "tyk-pump.pg_connection_string_secret_name" . }}
                 key: {{ include "tyk-pump.pg_connection_string_secret_key" . }}
           {{ end }}
+          {{ if has "postgres-pump" .Values.pump.backend }}
+          - name: TYK_PMP_PUMPS_SQL_TYPE
+            value: "sql"
+          - name: TYK_PMP_PUMPS_SQL_META_TYPE
```

Comment:
Should we name it `sql` pump instead of postgres-pump?
As it can be used for postgres and sqlite?
Same for postgres-aggregate

Reply:
I was also thinking the same think but decided to go with `postgres` as per https://github.com/TykTechnologies/tyk-charts/pull/118#discussion_r1352331941

Reply:
Makes sense.
Can you use `Postgres/Postgres-Aggregate Pump` instead of `SQL/SQL-Aggregate pump` in the readme too

Reply:
I updated docs in 58d6bccdf5978254ba03bd99ce4846d515647986. can you please check again?